frontends/ol-search-ui/src/util.tsx

```diff
@@ -75,7 +75,7 @@ const compareRuns = (
 const findBestRun = (
   runs: LearningResourceRun[]
 ): LearningResourceRun | undefined => {
-  const dated = runs.filter(run => run.best_start_date && run.best_end_date)
+  const dated = runs.filter(run => run.start_date && run.end_date)
```

Comment:
I think doing this instead would make it more consistent with `ExpandedLearningResourceDisplay.tsx`:

```javascript
  const startDate =
    hasCertificate && bestAvailableRun ?
      getStartDate(resource.platform ?? "", bestAvailableRun).format(DISPLAY_DATE_FORMAT) :
      null
```

Reply:
you're not wrong - that better solves the "and what if there's none" problem